decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -59,8 +59,6 @@
     github_handler { Faker::Hipster.word }
     sequence(:host) { |n| "#{n}.lvh.me" }
     description { Decidim::Faker::Localized.wrapped("<p>", "</p>") { Decidim::Faker::Localized.sentence(2) } }
-    welcome_text { Decidim::Faker::Localized.wrapped("<p>", "</p>") { Decidim::Faker::Localized.sentence(2) } }
-    homepage_image { Decidim::Dev.test_file("city.jpeg", "image/jpeg") }
```

Comment:
Can't we create content blocks at the factory? Or maybe we shouldn't?

Reply:
We can, but I don't really think it's needed in most cases